src/NRediSearch/Client.cs

```diff
@@ -486,22 +587,6 @@ public async Task<bool> DropIndexAsync()
             return (string)await _db.ExecuteAsync("FT.DROP", _boxedIndexName).ConfigureAwait(false) == "OK";
         }
 
-        /// <summary>
-        /// Optimize memory consumption of the index by removing extra saved capacity. This does not affect speed
-        /// </summary>
-        public long OptimizeIndex()
```

Comment:
what is the nature of the change here? does this no longer exist in the API? I would suggest:

``` c#
[Obsolete]
public long OptimizeIndex() => -1;
```

Reply:
Yes, apparently all the optimization is done behind the curtains without user intervention required.

Reply:
no problem with that; my concern is the library break - I'd rather deprecate it properly so people are directed to stop using it, rather than getting a `MissingMethodException`.

Reply:
Sounds good.

I added OptimizeIndex and OptimizeIndexAsync back in, they now return `default` and are marked as obsolete.